pyproject.toml

```diff
@@ -47,6 +47,9 @@ dev = [
     "responses",
     "ruff",
 ]
+configuration = [
+    "django-configurations>=2.5.1",
+]
```

Comment:
Once your branch rebased on the main one, you can update the `all` optional dependencies to add this one

```
all=[
    "django-lasuite[malware_detection]",
    "django-lasuite[configuration]",
]
```

And then your tests should work. The `all` is a requirement for `dev`